RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -372,39 +372,43 @@
 
 
 # Final sequence
-InitialStep = cms.Sequence(initialStepSeedLayers*
-                           initialStepTrackingRegions*
-                           initialStepHitDoublets*
-                           initialStepHitTriplets*
-                           initialStepSeeds*
-                           initialStepTrackCandidates*
-                           initialStepTracks*
-                           firstStepPrimaryVerticesUnsorted*
-                           initialStepTrackRefsForJets*
-                           caloJetsForTrk*
-                           firstStepPrimaryVertices*
-                           initialStepClassifier1*initialStepClassifier2*initialStepClassifier3*
-                           initialStep)
-_InitialStep_LowPU = InitialStep.copyAndExclude([firstStepPrimaryVerticesUnsorted, initialStepTrackRefsForJets, caloJetsForTrk, firstStepPrimaryVertices, initialStepClassifier1, initialStepClassifier2, initialStepClassifier3])
-_InitialStep_LowPU.replace(initialStep, initialStepSelector)
-trackingLowPU.toReplaceWith(InitialStep, _InitialStep_LowPU)
-_InitialStep_Phase1QuadProp = InitialStep.copyAndExclude([initialStepClassifier2, initialStepClassifier3])
-_InitialStep_Phase1QuadProp.replace(initialStepHitTriplets, initialStepHitTriplets+initialStepHitQuadruplets)
-trackingPhase1QuadProp.toReplaceWith(InitialStep, _InitialStep_Phase1QuadProp)
-_InitialStep_Phase1 = _InitialStep_Phase1QuadProp.copyAndExclude([initialStepHitTriplets])
-trackingPhase1.toReplaceWith(InitialStep, _InitialStep_Phase1)
-_InitialStep_trackingPhase2 = InitialStep.copyAndExclude([initialStepClassifier1, initialStepClassifier2, initialStepClassifier3])
-_InitialStep_trackingPhase2.replace(initialStepHitTriplets, initialStepHitQuadruplets)
-_InitialStep_trackingPhase2.replace(initialStep, initialStepSelector)
-trackingPhase2PU140.toReplaceWith(InitialStep, _InitialStep_trackingPhase2)
+InitialStepTask = cms.Task(initialStepSeedLayers,
+                           initialStepTrackingRegions,
+                           initialStepHitDoublets,
+                           initialStepHitTriplets,
+                           initialStepSeeds,
+                           initialStepTrackCandidates,
+                           initialStepTracks,
+                           firstStepPrimaryVerticesUnsorted,
+                           initialStepTrackRefsForJets,
+                           firstStepPrimaryVertices,
+                           initialStepClassifier1,initialStepClassifier2,initialStepClassifier3,
+                           initialStep,caloJetsForTrkTask)
+InitialStep = cms.Sequence(InitialStepTask)
+
+_InitialStepTask_LowPU = InitialStepTask.copyAndExclude([caloJetsForTrkTask,firstStepPrimaryVerticesUnsorted, initialStepTrackRefsForJets, firstStepPrimaryVertices, initialStepClassifier1, initialStepClassifier2, initialStepClassifier3])
```

Comment:
the order in of the arguments has changed.
Was it intentional?
It's just harder to make a comparison

Reply:
It probably got changed since I started edits.